Cargo.toml

```diff
@@ -10,12 +10,14 @@ license-file = "LICENSE"
 
 [dependencies]
 anyhow = "1.0.80"
-num-traits = "0.2.17"
+fast_image_resize = "3.0.4"
+gst = { version = "0.22.2", package = "gstreamer" }
+gst-app = { version = "0.22.0", package = "gstreamer-app" }
 image = { version = "0.25.0" }
-turbojpeg = {version = "1.0.0"}
 memmap2 = "0.9.4"
 ndarray = { version = "0.15.6", features = ["rayon"] }
-fast_image_resize = "3.0.4"
+num-traits = "0.2.17"
+turbojpeg = {version = "1.0.0"}
```

Comment:
might be worth considering putting different image file format supports behind feature flags so one can opt out of some of these dependencies (jpeg would be a good candidate for a default feature ofc)